gdnative-core/src/variant_array.rs

```diff
@@ -290,33 +542,45 @@ impl<T: ToVariant> Extend<T> for VariantArray {
 }
 
 godot_test!(test_array {
+    dbg!();
```

Comment:
Leftover `dbg`s! I guess we can deny `dbg_macro` and similar lints in CI only with command line flags? i.e. `cargo clippy -- -D clippy::dbg_macro -D clippy::todo -D clippy::unimplemented`?

Reply:
Somehow `-D clippy::dbg_macro` didn't work on my machine even with `--all-features`, I guess it only checks the code before macro expansion (which makes sense)? But it should be a good addition nevertheless.

Reply:
I haven't tried that but I removed those `dbg!()` invocations. Maybe we can try to add those flags in a separate PR about the CI setup (I still want to enable caching for `cargo-dinghy`)